evap/staff/tests/test_views.py

```diff
@@ -60,16 +81,17 @@ def test_deletes_users(self):
 
 class TestSemesterExportView(ViewTest):
     url = '/staff/semester/1/export'
-    users = ['staff']
+    test_users = ['staff']
```

Comment:
it didn't. since "test_users" was not overridden and thus empty, the test simply did nothing. there are no other tests with this bug.